.changeset/unlucky-ads-film.md

```diff
@@ -0,0 +1,11 @@
+---
+"@balancer/sdk": minor
+---
+nt
+Adds Balancer's V3 CreatePool functionality for Weighted Pools;
+Adds tests for the V3's Create Weighted Pool;
+Adds example for V3's Create Weighted Pool;
+Renames factories references to differ by balancer version instead of pool version, or no differentiation;
+Adapts createPoolHelper to work with V3;
+Adds TokenType enum;
+Add another validation step for V3's Create Weighted Pool input(rateProvider cannot be zero if tokenType is not STANDARD);
```

Comment:
These are actually release notes, so you could/should be more concise.
> Add V3 CreatePool functionality for Weighted Pools

This one likely covers all items on this list.
If you really want to highlight something else, feel free to add another line or two, otherwise you can remove most of them.